internal/errors.go

```diff
@@ -7,13 +7,14 @@ import (
 	"github.com/getsentry/sentry-go"
 	"os"
 	"runtime"
+	"time"
 
 	"github.com/rs/zerolog"
 )
 
 var logger = zerolog.New(os.Stdout).With().Timestamp().Logger().Output(zerolog.ConsoleWriter{
 	Out:        os.Stderr,
-	TimeFormat: "15:04:05",
+	TimeFormat: time.RFC3339,
```

Comment:
Aside: Is there a way to force the output to always be UTC, even if the process is running in a non-UTC locale?

(No worries if not)